diff --git a/Sources/KeyboardShortcuts/Utilities.swift b/Sources/KeyboardShortcuts/Utilities.swift
--- a/Sources/KeyboardShortcuts/Utilities.swift
+++ b/Sources/KeyboardShortcuts/Utilities.swift
@@ -4,0 +5,16 @@ import Carbon.HIToolbox
+
+private extension Bundle {
+	static let keyboardShortcutsResources: Bundle = {
+		guard
+			let bundleURL = Bundle.main.url(
+				forResource: "KeyboardShortcuts_KeyboardShortcuts",
+				withExtension: "bundle"
+			),
+			let bundle = Bundle(url: bundleURL)
+		else {
+			return .module
+		}
+
+		return bundle
+	}()
+}
@@ -12 +28 @@ extension String {
-		NSLocalizedString(self, bundle: .module, comment: self)
+		NSLocalizedString(self, bundle: .keyboardShortcutsResources, comment: self)
